### ISSUE TITLE
Test::Alien::Diag::alien_diag() testing for explicit method

### ISSUE THREAD

**Issue by michal-josef-spacek** [PerlAlien/Alien-Build#365]
There is a situation with Alien::Brotli, which deliver new exe() subroutine, which returns the path to brotli tool.
Should be good to present this result in alien_diag() method.

My proposal is to add an array of method names to alien_diag() or create another testing subroutine to handle these explicit methods.

**Comment by plicease** [comment]
This seems reasonable.  `alien_diag` currently takes an array of alien  names.

```perl
alien_diag('Alien::libfoo', 'Alien::libbar');
```

I think what we want is to recognize an optional options hash at the end.

```perl
alien_diag('Alien::libfoo', 'Alien::libbar', { properties => [ 'exe', ... ]);
```

**Comment by plicease** [comment]
Proposed change was included in AB 2.68

**Comment by michal-josef-spacek** [comment]
Great, thank you.
I created PR for Alien::Brotli (https://github.com/robrwo/perl-Alien-Brotli/pull/8)
